src/com/hms_networks/americas/sc/historicaldata/HistoricalDataConstants.java

```diff
@@ -50,6 +50,9 @@
   /** Name of EBD string export call file */
```

Comment:
commit message should get updated to 

```
Adds automated recovery from corrupted file. Uses a second
time tracker file to prevent lost data due to file corruption. Uses
current time in the case both files are lost.
```